### PR TITLE
Removed former members | Added Ricardo Quiroz | Added Team Member Tags

### PR DESCRIPTION
**Pull request by quirozj**
-The tags aren't the best (I need to get better at CSS) or very helpful, so I understand if someone doesn't like them
-No photo for Ricardo Quiroz yet
-Andrew told me which members to remove

### CODE REVIEW

**Comment by quirozj** [comment]
Woah I think I messed that up really bad I'm so sorry

File: _pages/team.html [hunk 1]
```diff
@@ -9,125 +9,126 @@
...
             <h5>Kevin Zhang</h5>
-            <p>Kevin is a new member of the team.</p>
+            <p>Kevin is a mechanic on the team.</p>
+            <p2>🔧Mechanic🔧</p2>
         </div>
         <div>
             <img src="/images/team/placeholder.png">
             <h5>Rachel Danckaert</h5>
-            <p>Rachel is a new member of the team.</p>
+            <p>Rachel was previously outreach and is now a programmer.</p>
```
Comment (ErikBoesen) [inline _pages/team.html]: I think it might make more sense to say "Rachel previously worked on outreach"

File: _pages/team.html [hunk 2]
```diff
@@ -9,125 +9,126 @@
...
+        <div>
+             <img src="/images/team/julian-quiroz.png">
+             <h5>Julian Quiroz </h5>
+             <p>Julian Quiroz is a new member of the team and is a programmer.</p>
+             <p2>🖥️Programmer🖥️</p2>
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <h5>Ricardo Quiroz </h5>
+            <p>Ricardo Quiroz is a new member of the team and is apart of the outreach team.</p>
```
Comment (ErikBoesen) [inline _pages/team.html]: *a part

File: _pages/team.html [hunk 3]
```diff
@@ -9,125 +9,126 @@
...
-        <div>
-            <img src="/images/team/erik-boesen.png">
-            <h5>Erik Boesen</h5>
-            <p>Erik is President of 1418 as well as Programming Captain. Aside from leading development of all our team's software projects, he works on team public relations and with the Virginia House of Delegates to pass legislation aiding FIRST teams in economically disadvantaged communities. He was placed on the FIRST Dean's List in 2018.</p>
-        </div>
         <div>
             <img src="/images/team/ben-updike.png">
             <h5>Ben Updike</h5>
             <p>Ben is Mechanics Captain and leads the design, building, and maintenance of the robot. During competitions, he is one of the two drivers and operates the robot's various mechanisms. Last season, he served as team technician and battery manager.</p>
+            <p2>🔧Mechanic🔧</p2>
```
[the remark this answers is not in the document]
Comment (ErikBoesen) [reply _pages/team.html]: Rather than p2, perhaps try a regular `<p>` or a `<div>` with a `class="tag"` attribute and style based on that?

**Comment by ErikBoesen** [comment]
Also, title commits in the imperative tense, so "Remove..." rather than "Removed..."

**Pull request merged**